packages/now-python/now_init.py

```diff
@@ -9,6 +9,7 @@
 __now_spec = util.spec_from_file_location("__NOW_HANDLER_MODULE_NAME", "./__NOW_HANDLER_ENTRYPOINT")
 __now_module = util.module_from_spec(__now_spec)
 __now_spec.loader.exec_module(__now_module)
+sys.modules["__NOW_HANDLER_MODULE_NAME"] = __now_module
```

Comment:
Thanks for this PR!

It looks like this is failing because `sys` is undefined.

You probably need to add `import sys` on line 5 and remove the import on line 87.

Also, if you could change the stackoverflow link to the official python docs that would be superb 👌